test/1_stdlib/Inputs/ProcessStressTest/ProcessStressTest.c

```diff
@@ -0,0 +1,48 @@
+#include <CoreFoundation/CoreFoundation.h>
+#include <dispatch/dispatch.h>
+#include "ProcessStressTest.h"
+#include <pthread.h>
+#include <assert.h>
+
+static pthread_mutex_t mutex = PTHREAD_MUTEX_INITIALIZER;
+static pthread_mutex_t waiterMutex = PTHREAD_MUTEX_INITIALIZER;
+
+static void run_process_stress_test(void) {
+  dispatch_queue_t queue = dispatch_queue_create(nil, DISPATCH_QUEUE_CONCURRENT);
+  dispatch_group_t group = dispatch_group_create();
+  pthread_mutex_init(&mutex, NULL);
+  pthread_mutex_init(&waiterMutex, NULL);
+
+  CFMutableSetRef results = CFSetCreateMutable(NULL, 0, &kCFTypeSetCallBacks);
+  pthread_mutex_lock(&waiterMutex);
+  for (uint32_t i = 0; i < 1000; ++i) {
+    dispatch_group_enter(group);
+    dispatch_async(queue, ^{
```

Comment:
I think you can use [`dispatch_group_async`](https://developer.apple.com/reference/dispatch/1453084-dispatch_group_async) for this.